src/components/modules/Search.js

```diff
@@ -36,11 +36,18 @@ const SearchLogic = () => {
 
     return (
         <div className="SearchLogic">
-            <form onSubmit={getSearch} className="Search-form">
-                <input className="search-bar" type="text" value={search} onChange={updateSearch} />
-                <button className="search-button"
-                    type="submit">Search</button>
-            </form>
+            <FormWrapper>
+                <form onSubmit={getSearch} className="SearchForm">
+                    <input className="search-bar" type="text" value={search} onChange={updateSearch} />
+                    <button className="search-button"
+                        type="submit">Search</button>
+                </form>
+                <form onSubmit={getSearch} className="FilterForm">
+                    <button className="search-button-filter" onClick={() => setQuery('low-carb')}>Low Carb</button>
+                    <button className="search-button-filter" onClick={() => setQuery('keto')}>Keto</button>
+                    <button className="search-button-filter" onClick={() => setQuery('paleo')}>Paleo</button>
+                </form>
+            </FormWrapper>
             {recipes.map(recipe => (
```

Comment:
you could use destructuring assingment here to avoid writing 'recipe' several times in the following lines